sirepo/tornado.py

```diff
@@ -6,8 +6,30 @@
 """
 from pykern.pkdebug import pkdlog, pkdexc
 import sirepo.util
+import tornado.locks
 import tornado.queues
 
+class Event(tornado.locks.Event):
+    class _OrderedSet:
```

Comment:
I think this could subclass list, and then all you'd need to do was implement:

````py
def add(self, value):
    self.append(value)
```

I would also call this class _OrderedWaiters, since it's not a generic ordered set, and doesn't need to be.

Reply:
https://github.com/radiasoft/sirepo/pull/2731/commits/6960d7d6b3672704a51babc454d7c6b07a162c87